### ISSUE TITLE
I get a fixed mAP after training for a while  

### ISSUE THREAD

**Issue by HGamal11** [fizyr/keras-retinanet#875]
I'm training the model on my own dataset and the mAP for the test set is not going up or down .. if the model was overfitting the mAP would change for the test data but it's not .. just the same number no matter how much I train further
The loss moves around some number and doesn't change much 
I tried with two different datasets and both did the same thing after training for a while

**Comment by hgaiser** [comment]
How big is your test set? What is the mAP values ?

**Comment by HGamal11** [comment]
3000 images .. the mAP is 0.6358

**Comment by HGamal11** [comment]
If I use snapshot to resume training does the LR starts from 1e-5 or does it keep the value it reached on the last epoch ?

**Comment by hgaiser** [comment]
If you use snapshot it resumes where it left off, if you use --weights it
re-initializes the optimiser.

**Comment by HGamal11** [comment]
That was the problem I used --snapshot not --weights and the learning rate I reached was very low .. the loss started changing again .. Thanks
